scrapper_config.json

```diff
@@ -1,4 +1,6 @@
 {
-    "seed_urls": [],
-    "total_articles_to_find_and_parse": 0
+    "seed_urls": ["https://rjano.ruslang.ru/ru/archive/2001-1", "https://rjano.ruslang.ru/ru/archive/2001-2", "https://rjano.ruslang.ru/ru/archive/2002-1",
```

Comment:
that is a good list of seed_urls. however, I would recommend to get more recent volumes, like this: https://rjano.ruslang.ru/ru/archive/2021-2